src/main/scala/thurloe/service/ThurloeService.scala

```diff
@@ -158,5 +159,23 @@ trait ThurloeService extends HttpService {
     }
   }
 
-  val keyValuePairRoutes = getRoute ~ getAllRoute ~ queryRoute ~ setRoute ~ deleteRoute
+  val getStatus = path(ThurloePrefix / "status") {
```

Comment:
it looks like this endpoint overlaps with the /thurloe/{userId} endpoint for retrieving user key/values. getting rid of ThurloePrefix would fix it